ls1intum/Artemis
# Bugfix/re evaluate quiz score misaligned #924

### Checklist

### Motivation and Context


see issue #924
### Description

Modified the colon symbol style by removing `display:inline-flex;` , because it does not make flex items display inline,but the flex container.
### Steps for Testing


1. Go to 'Course Administration'
2. Click on 'Re-evaluate' on a Quiz exercise
3. See that the score correctly aligned

### Screenshots


![image](https://user-images.githubusercontent.com/50635005/67149243-3be88600-f2a9-11e9-86e8-48d09a004cc8.png)
